### ISSUE TITLE
circle results in "no data found" when GRanges has names

### ISSUE THREAD

**Issue by seandavi** [tengfei/ggbio#84]
If the GRanges has names associated with the rows, I get `no data found`.  Version 1.22.0 of ggbio and 2.2.0 of ggplot.

**Comment by lawremi** [comment]
Is there any code for reproducing this?

**Comment by seandavi** [comment]
```
> library(ggbio)
> data("CRC", package = "biovizBase")
> mut.gr
*** output flushed ***
> p <- ggbio() + circle(mut.gr, geom = "rect", color = "steelblue")
> names(mut.gr) = rep('abc',length(mut.gr))
> p <- ggbio() + circle(mut.gr, geom = "rect", color = "steelblue")
Error in ggbio() + circle(mut.gr, geom = "rect", color = "steelblue") : 
  no data found
```

**Comment by lawremi** [comment]
Thanks, I fixed it in 1.23.3.

**Comment by seandavi** [comment]
Thx, @lawremi.
